astroid/nodes/node_classes.py

```diff
@@ -2705,16 +2705,19 @@ class ParamSpec(_base_nodes.AssignTypeNode):
     <ParamSpec l.1 at 0x7f23b2e4e198>
     """
 
+    _astroid_fields = ("name",)
```

Comment:
Thanks, I missed these because I went back and forth on whether this should be a `str` or an `AssignName` until I saw the references for the similar issue with `MatchStar` and friends pointing to https://github.com/python/cpython/issues/88160.